components/ui/icons/src/main/res/drawable/mozac_ic_lock.xml

```diff
@@ -2,12 +2,6 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="24dp"
-        android:height="24dp"
-        android:viewportWidth="24.0"
-        android:viewportHeight="24.0">
-    <path
-        android:fillColor="#ffffff"
-        android:pathData="M17.25,11H17V8A5,5 0,0 0,7 8v3H6.75A1.814,1.814 0,0 0,5 12.871v6.257A1.814,1.814 0,0 0,6.75 21h10.5A1.814,1.814 0,0 0,19 19.129V12.871A1.814,1.814 0,0 0,17.25 11ZM9,7.739A2.739,2.739 0,0 1,11.739 5h0.523A2.739,2.739 0,0 1,15 7.739V11H9Z"/>
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
+    <path android:fillColor="#F9F9FA" android:fillType="evenOdd" android:pathData="M9 8c0-1.657 1.343-3 3-3s3 1.343 3 3v3H9V8zm-2 3V8c0-2.761 2.239-5 5-5s5 2.239 5 5v3c1.105 0 2 0.895 2 2v6c0 1.105-0.895 2-2 2H7c-1.105 0-2-0.895-2-2v-6c0-1.105 0.895-2 2-2z"/>
```

Comment:
Those new ones use `evenOdd` which isn't supported on all versions of Android. Where is that coming from?

Reply:
Hm I think that was from avocado

Reply:
Oh, that's weird. Sometimes it's enough to remove the keyword and it will still render fine.

Definitely test Focus on an API 21 device or similar to make sure it renders fine there too.